storage-proofs-porep/src/stacked/vanilla/create_label/multi.rs

```diff
@@ -77,13 +77,13 @@ fn fill_buffer(
     // Node 5 (prev node) will always be missing, and there tend to be
     // frequent close references.
     if cur_node > MIN_BASE_PARENT_NODE {
-        // Mark base parent 5 as missing
+        // Mark base parent 0 as missing
         // base_parent_missing.set_all(0x20);
-        base_parent_missing.set(5);
+        base_parent_missing.set(0);
```

Comment:
Good catch. It looks like when the graph was changed (conditionally, depending on version), this code was not updated to also be conditional. So the old graph is always being used here. To make this fully correct now, it should be conditional here now too. The current formulation for the decision can be seen here: https://github.com/filecoin-project/rust-fil-proofs/blob/00cb80440d14dd760805c14db597a306d4433fb2/storage-proofs-core/src/drgraph.rs#L166-L169.

Reply:
You are right! The current implemention doesn't has a good compatibility with different versions of the graph (which it should be).I'll try to improve this PR to solve this quesion.